docs/site/TEMP-TEXT-FILES/guide.txt

```diff
@@ -1,31 +1,34 @@
 Note that the exact appearance in screenshots may change as the tool is enhanced over time and based on your vetter level.
-💡 Helpful Tips 
-Please read the home page of the Translation Guidelines before starting your data contribution. 
+💡 Helpful Tips
+Please read the home page of the Translation Guidelines before starting your data contribution.
 If you experience a Loading... problem with the Survey Tool, try clearing your browser cache. See Reloading JavaScript.
-Browser support for Survey Tool includes the latest versions of Edge, Safari, Chrome, and Firefox. 
+Browser support for Survey Tool includes the latest versions of Edge, Safari, Chrome, and Firefox.
 Use Reports at the beginning to review the data in your language in a holistic manner for Date & time, Zones, and Numbers.
 Capitalization: Translations should be what is most appropriate for ”middle-of-sentence” use. So, for example, if your language normally doesn't spell languages with a capital letter, then you shouldn’t do that here. Please see Capitalization for more details.
 Plurals: For important information regarding the use of plural forms for your language, please read Plurals.
 Vetting Phase
 At a point towards the end of Survey Tool period, the Technical Committee will change the survey tool to "Vetting Mode". In Vetting Mode, submitting new data/translations is no longer possible, but you can still change your votes and participate in the forum. (The exception is that you can submit new data if the currently winning value has generated an error or a warning.)
 Login and Import of old votes
-Go to http://st.unicode.org/cldr-apps/survey/ and log in.
-Import of old votes is automatically handled when you log in for all your votes matching the latest released data. If you have voted previously, upon first log-in, you will see a message showing the number of your votes that matched the currently winning votes that have been auto-imported. 
-Import of your old votes will take some time, but this will be done at 1st log-in only.
-Import old votes manually. You can still import your old voted data that DO NOT match the last released data. 
-Go to the  ☰ Menu, look under My Votes, and choose Import Old Votes.  
+Go to http://st.unicode.org/cldr-apps/survey/ and log in. 
+Import of old votes is automatically handled when you log in for all your votes matching the latest released data. If you have voted previously, upon first log-in, you will see a message showing the number of your votes that matched the currently winning votes that have been auto-imported.
+Import of your old votes will take some time, but this will be done at 1st log-in only. 
+Import old votes manually. You can still import your old voted data that DO NOT match the last released data.
+Go to the G☰ Menu, look under My Votes, and choose Import Old V****otes. 
```

Comment:
This also looks like an error.

Reply:
Still a problem. There is no 
> Go to the **G☰ Menu**

There is no "G☰" menu. You need to remove the "G". Here is what it looks like; in this image, the symbol after "Vetting"

![Screenshot 2024-06-19 at 16 43 23](https://github.com/unicode-org/cldr/assets/9613872/a0b73ec5-1380-4b7a-80cf-8a1797b1fc43)

https://st.unicode.org/cldr-apps/v#/de//